projects/plugin/src/plugin/components/debrid-files/debrid-files.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+
+import { HttpClient } from '@angular/common/http';
+
+import { DebridAccountService } from '../../services/debrid-account.service';
+import { OpenSourceService } from '../../services/open-source.service';
+
+@Component({
+  selector: 'wk-debrid-files',
+  templateUrl: './debrid-files.component.html',
+  styleUrls: ['./debrid-files.component.scss']
+})
+export class DebridFilesComponent implements OnInit {
+  private key = '';
+
+  private base_link = 'https://www.premiumize.me/api';
+
+  private api_folderAll = '/folder/list';
+
+  public folder;
+
+  constructor(private http: HttpClient, private debridAccountService: DebridAccountService, private openSourceService: OpenSourceService) {}
+
+  ngOnInit() {
+    this.debridAccountService
+      .getPremiumizeSettings()
+      .then((settings) => {
+        this.key = settings.apiKey;
+      })
+      .then(() => {
+        this.listAll('');
+      });
+  }
+
+  public async listAll(folderID) {
+    let data;
+
+    if (folderID !== '') {
+      data = await this.http.get(this.base_link + this.api_folderAll + '?id=' + folderID + '&apikey=' + this.key).toPromise();
```

Comment:
There is a PM service, you don't need to retrieve the api key. Create form + dto for all new API routes in https://github.com/wako-unofficial-addons/helios/tree/master/projects/plugin/src/plugin/services/premiumize/forms.

You can see an example of how to use services `forms` here https://github.com/wako-unofficial-addons/helios/blob/master/projects/plugin/src/plugin/components/debrid-account/debrid-account.component.ts#L102

For instance here in `src/plugin/services/premiumize/forms` create directory named `folder`. Inside create a form called `premiumize-folder-list.form.ts` which should contains: 
```ts
import { PremiumizeApiService } from '../../services/premiumize-api.service';
import { PremiumizeTransferCreateDto } from '../../dtos/folder/premiumize-folder-list.dto';

export class PremiumizeFolderListForm {
  static submit(folderId?: string) {
    return PremiumizeApiService.get<PremiumizeFolderListDto>('/folder/list', {
      id: folderId
    });
  }
}
```
Create the file PremiumizeFolderListDto (dtos/folder/premiumize-folder-list.dto) which will represent the data returned by the API